include/multipass/platform.h

```diff
@@ -86,6 +86,12 @@ std::function<int()> make_quit_watchdog(); // call while single-threaded; call r
 
 std::string reinterpret_interface_id(const std::string& ux_id); // give platforms a chance to reinterpret network IDs
 
+std::string read_os_release(const QString& path = "/var/lib/snapd/hostfs/etc/os-release");
```

Comment:
Reading of `os-release` should fall back to `(/var/lib/snapd/hostfs)/usr/lib/os-release` when `(...)/etc/os-release` can't be read. See `man 5 os-release`.